build.gradle.kts

```diff
@@ -87,7 +87,7 @@ gradlePlugin {
     create("dockerPublishPlugin") {
       id = "de.europace.docker-publish"
       displayName = "Docker Publish Plugin"
-      description = "Adds tasks to create and publish a docker image to docker hub"
+      description = "Adds tasks to create and publish a Docker image to a registry"
```

Comment:
hmm can we actually change the registry at the moment? I thought at the moment this is hard coded?

Reply:
The registry may be selected by configuring an organisation like `docker-registry:5000/organisation-name`. We might add an explicit configuration property later, but that's more a nice to have.